railties/lib/rails/generators/app_base.rb

```diff
@@ -15,6 +15,8 @@ class AppBase < Base # :nodoc:
       include Database
       include AppName
 
+      NODE_TLS_VERSION = "18.15.0"
```

Comment:
@byroot Is it typo?

Reply:
🤦 yes it is.

Reply:
Fixed in 2d6ce962b0dee0d1bb9f0c9535d15006f3963b9f, thank you for the ping.